core/templates/dev/head/domain/exploration/SubtitledHtmlObjectFactorySpec.js

```diff
@@ -27,12 +27,13 @@ describe('SubtitledHtml object factory', function() {
       shof = $injector.get('SubtitledHtmlObjectFactory');
       subtitledHtml = shof.createFromBackendDict({
         html: '<p>some html</p>',
-        audio_translations: [{
-          language_code: 'en',
-          filename: 'a.mp3',
-          file_size_bytes: 20,
-          needs_update: false
-        }]
+        audio_translations: {
+          en: {
```

Comment:
Why is it that for states we include the state name itself both as a key and a value? I feel like maybe that pattern would be useful here? I think it may have been done for binding purposes before, but I'm not sure.